### PR TITLE
What if code added

### PR DESCRIPTION
**Pull request by avneet006**
Code added for what if Scenario

### CODE REVIEW

File: twin4build/__init__.py [hunk 1]
```diff
@@ -1,37 +1,37 @@
-from twin4build.model.model import Model
-from twin4build.simulator.simulator import Simulator
-from twin4build.monitor.monitor import Monitor
-from twin4build.estimator.estimator import Estimator
-from twin4build.evaluator.evaluator import Evaluator
+# from twin4build.model.model import Model
+# from twin4build.simulator.simulator import Simulator
+# from twin4build.monitor.monitor import Monitor
+# from twin4build.estimator.estimator import Estimator
+# from twin4build.evaluator.evaluator import Evaluator
```
Comment (SebsCubs) [inline twin4build/__init__.py:5]: Hi @avneet006 Is there any reason in special to comment out these imports?
It breaks the short import statements that can now be used in models (i.e. tb.Model(id,saveSimulationResult), or tb.SensorSystem, etc).

**Review by SebsCubs — COMMENTED** [review]
Please check out these commented out lines, thanks.

**Comment by avneet006** [comment]
Hello Sebastian,

We have DMI forecast data fetching component, to run that component we have to comment out these lines you can uncomment those line if required.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by SebsCubs** [comment]
I think we should find a way of having both features working. Is there a way of running the DMI component without commenting these lines?

**Pull request merged**